packages/common/editors/__tests__/CheckboxEditor.spec.js

```diff
@@ -1,27 +1,45 @@
 const React          = require('react');
 const ReactDOM = require('react-dom');
 const TestUtils      = require('react-dom/test-utils');
+const { mount } = require('enzyme')
+const sinon = require('sinon')
 const CheckboxEditor = require('../CheckboxEditor');
-const { mount } = require('enzyme');
 
 describe('CheckboxEditor', () => {
   let component;
   let componentWrapper;
+  let consoleErrorStub;
   let testColumn = {
     key: 'columnKey',
     onCellChange: function() {}
   };
 
   describe('Basic tests', () => {
+    afterEach(() => {
+      consoleErrorStub.restore()
+    })
+
     beforeEach(() => {
       spyOn(testColumn, 'onCellChange');
+      consoleErrorStub = sinon.stub(console, 'error')
```

Comment:
Here we stub console.error to intercept any possible messages passed to it. Because the code change being tested actually prevents the messages from being raised to console.error, we will never see them. This "absence of an error message" is hard to test without changing the way props are passed into this component, which feels like more than is necessary for this minor change.

If you're curious or otherwise want to ensure this test does what it claims to do, remove the readOnly flag from line 25 of packages/common/editors/CheckboxEditor.js and run the tests.